server/src/test/java/io/crate/planner/optimizer/rule/RewriteFilterOnCrossJoinToInnerJoinTest.java

```diff
@@ -116,17 +116,17 @@ public void test_rewrite_equ_join_filter_on_top_of_cross_join_to_inner_join_on_s
 
     @Test
     public void test_rewrite_equ_join_filter_on_top_of_nested_cross_join_to_inner_join() {
```

Comment:
since this is now just a simpler version of the next test, just without the constant filter, should we maybe just remove it?

Reply:
https://github.com/crate/crate/pull/17090#discussion_r1865771485